### ISSUE TITLE
Include supports-color as dependency

### ISSUE THREAD

**Issue by Qix-** [visionmedia/debug#612]
Right now, we enable 256 colors if you have `supports-color` as a dependency. A few individuals have mentioned it should be included in the project dependencies, which sounds ideal.

The reason why we didn't (initially) do so is because we didn't want a minor/patch bump to introduce yet another dependency.

Thoughts?

**Comment by Jezorko** [comment]
I think it's better to give users a choice. Not everyone needs fancy colors. Also, how to add them is well explained in the README.

**Comment by Qix-** [comment]
@jezorko Yeah I'm back and forth, but I mostly agree. A few times, though, people have complained about it - hence this ticket.
